website/cue/reference/components/sinks/aws_s3.cue

```diff
@@ -368,7 +368,7 @@ components: sinks: aws_s3: components._aws & {
 
 			policies: [
 				{
-					_action: "HeadBucket"
+					_action: "ListBucket"
```

Comment:
I think the cue file needs to be updated too.

Reply:
👀

Reply:
I am definitely awake and functioning 100% :rofl: